CONTRIBUTING.md

```diff
@@ -56,13 +56,35 @@ large_files:
     url: "url-of-model.ts"
 ```
 
+### Preferred Files and Keys
+
+In order to be compatible with other apps such as MONAI FL, MONAI deploy and MONAI Label, except the MONAI Bundle Specification, a bundle in MONAI Model Zoo should contain the necessary file `configs/metadata.json` and the following preferred files:
```

Comment:
perhaps put a link of example here explaining the definition of the files and keys?

Reply:
Thanks @wyli , examples are added.